segments/hostname.sh

```diff
@@ -3,6 +3,10 @@
 
 TMUX_POWERLINE_SEG_HOSTNAME_FORMAT="${TMUX_POWERLINE_SEG_HOSTNAME_FORMAT:-short}"
 
+command_exists() {
```

Comment:
Thank you. Acctually I see that this function would be good to make available in all segments. Could you move it to here?

https://github.com/erikw/tmux-powerline/blob/main/lib/util.sh